content/en/blog/2024/v1-apis/index.md

```diff
@@ -0,0 +1,81 @@
+
+In Istio 1.22 we are happy to announce that a concerted effort has been made to graduate the below APIs to `v1`:
+* [Destination Rule](/docs/reference/config/networking/destination-rule/)
+* [Gateway](/docs/reference/config/networking/gateway/)
+* [Service Entry](/docs/reference/config/networking/service-entry/)
+* [Sidecar](/docs/reference/config/networking/sidecar/)
+* [Virtual Service](/docs/reference/config/networking/virtual-service/)
+* [Workload Entry](/docs/reference/config/networking/workload-entry/)
+* [Workload Group](/docs/reference/config/networking/workload-group/)
+* [Telemetry API](/docs/reference/config/telemetry/)*
+* [Peer Authentication](/docs/reference/config/security/peer_authentication/)
+
+## Telemetry API
+
+The `v1` Telemetry API is the only API that was promoted that had changes from its previous version. The following features weren’t promoted to `v1`:
+* `metrics.reportingInterval`
+    * Reporting interval allows configuration of the time between calls out to for metrics reporting. This currently only supports TCP metrics but we may use this for long duration HTTP streams in the future.
+    * At this time, Istio lacks usage data to support the need for this feature.
+* `accessLogging.filter`
+    * If specified, this filter will be used to select specific requests/connections for logging.
+    * This feature is based on a relatively new feature in Envoy, and Istio needs to further develop the use case and implementation before graduating it to `v1`.
+* `tracing.useRequestIdForTraceSampling`
+    * This value is true by default. The format of this Request ID is specific to Envoy, and if the Request ID generated by the proxy that receives user traffic first is not specific to Envoy, Envoy will break the trace because it cannot interpret the Request ID. By setting this value to false, we can prevent [Envoy from sampling based on the Request ID](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/tracing#trace-context-propagation).
+    * There is not a strong use case for making this configurable through the Telemetry API.
+
+When migrating Telemetry Resources to the `v1` Telemetry API, we strongly recommend removing the usage of the non-`v1` fields. Not doing this will not cause any disruptions, as Istio has chosen to maintain the same schema across API Versions to avoid unyielding API conversion issues. This is a known, complex issue with [Kubernetes API Versioning](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#overview), and the Gateway API has chosen to address this with [Release Channels](https://gateway-api.sigs.k8s.io/concepts/versioning/).
```

Comment:
"Not doing this will not cause any disruptions" - this is a double negative and a bit confusing. What are you advising users to do here and for what reason?

Reply:
```suggestion
```

I think we could actually just cut this entire paragraph - the more user-focused instructions below under "Using the v1 Istio APIs" feel sufficient for managing practical concerns, and getting into the "why" and "how" here feels a bit distracting for a primarily end-user audience.

Reply:
Nixed!